modules/naplicense/src/licenseservice.cpp

```diff
@@ -51,26 +51,44 @@ namespace nap
 	}
 
 
-	static bool rsaVerifyFile(const std::string& publicKey, const std::string& licenseFile, const std::string& signatureFile)
+	static CryptoPP::PK_Verifier* getVerifier(const std::string& publicKey, nap::SigningScheme signingScheme)
+	{
+		CryptoPP::StringSource pub_file(publicKey.c_str(), true, new CryptoPP::HexDecoder);
+		CryptoPP::PK_Verifier* verifier = nullptr;
+		if (signingScheme == nap::SigningScheme::RSASS_PKCS1v15_SHA1)
```

Comment:
We prefer switch statements over if else blocks. All cases must be handled, therefore `default` asserts and returns nullptr.